roles/ora-host/tasks/main.yml

```diff
@@ -90,6 +97,16 @@
     - oraclelinux_repo.stat.exists|bool
   tags: os-packages
 
+- name: Install Oracle required packages (base/non-rhui config) for OEL8
+  yum:
+    name: "{{ oracle_required_rpms_oel8 }}"
```

Comment:
oracle_required_rpms_el8 with the most recent commit of roles/ora-host/defaults/main.yml, right?

Reply:
Thanks, good catch !